modules/ROOT/pages/export-to-exchange-task.adoc

```diff
@@ -5,30 +5,23 @@ endif::[]
 
 Anypoint Studio allows you to connect to Exchange to publish your project as a new asset.
 
-. Right click the project folder on your Package Explorer View.
-. Select Anypoint Platform and then Publish to Exchange.
-+
-If you don't have your Anypoint Platform credentials configured in Anypoint Studio, you'll need to provide your log in information before continuing.
-. Select your Anypoint Platform user and business group.
-+
+. Right-click your project's folder on your Package Explorer view.
+. Select *Anypoint Platform* > *Publish to Exchange*.
+. Select your Anypoint Platform user and business group. +
 Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
```

Comment:
This is not an addition to the step but rather an alternative.

Consider whether adding a new account is a likely scenario for connecting to Exchange to publish your project as a new asset.